bundles/org.openhab.binding.miio/README.md

```diff
@@ -109,6 +109,7 @@ However, many devices share large similarities with existing devices.
 The binding allows to try/test if your new device is working with database files of older devices as well.
 
 There are 3 ways to get unsupported devices working:
+
```

Comment:
this miio readme.md is automatically created based on README.base.md
Hence the base file needs to be adjusted as well otherwise with the next update this change will be automatically overwritten.

Reply:
Thanks @marcelrv. Done.